src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -844,6 +848,18 @@ bool fuse_type_to_multiclass_nms(const std::shared_ptr<ov::Node>& node, const pr
     });
 }
 
+bool fuse_type_to_multinomial_v13(const std::shared_ptr<ov::Node>& node, const precisions_map& precisions) {
+    auto multinomial = ov::as_type_ptr<opset13::Multinomial>(node);
+    if (!multinomial) {
+        return false;
+    }
+
+    return update_type(0, node, precisions, [&](const element::Type& type) {
+        multinomial->set_convert_type(ov::element::i32);
```

Comment:
This should not be hardcoded - see my previous comment.
```suggestion
        multinomial->set_convert_type(type);
```